app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
Place each argument in 1 line

Reply:
Fix in all places

Reply:
If I place arguments in one line. Flake8 : E501 line too long